plugins/source/azure/policies/hipaa_hitrust_v9.2/network_controls.sql

```diff
@@ -19,7 +19,7 @@
 \ir ../queries/keyvault/vaults_with_no_service_endpoint.sql
 \set check_id '0866.09m3Organizational.1516 - 09.m - 1'
 \echo :check_id
-\ir ../queries/storage/accounts_with_not_restricted_asscess.sql
+\ir ../queries/storage/accounts_with_not_restricted_access.sql
```

Comment:
I feel like this should be either:
```suggestion
\ir ../queries/storage/accounts_with_unrestricted_access.sql
```
or
```
accounts_without_restricted_access
```

and probably the first

Reply:
Fixed